src/bot.js

```diff
@@ -15,41 +15,62 @@ const { version } = require('../package.json')
 const client = new Client()
 let guild, db
 
-process.title = `WAHtson ${version}`
-console.log(`WAHtson ${version}`)
-
-config
-    .load()
-    .then(() =>
-        sqlite.open({
-            filename: './database.sqlite',
-            driver: Database,
-        }),
-    )
-    .then(async _db => {
-        db = _db
-        await db.migrate()
-    })
-    .then(async () => {
-        config.get('bot_token', async token => {
-            await client.login(token)
-            return true
-        })
-    })
-    .catch(err => {
-        console.error(chalk.red(`error: ${err}`))
-        process.exit(1)
-    })
+const EventEmitter = require('events')
+
+let send
+
+module.exports = {
+    Wahtson: class Bot extends EventEmitter {
+        constructor(botOptions) {
+            super()
+            this.botOptions = botOptions || {}
+            this.version = version
+
+            send = async (a, b) => {
+                this.emit(a, b)
+            }
+            module.exports.send = async (a, b) => {
+                this.emit(a, b)
+            }
+            send('info', ['test', 'cyan'])
+        }
+        async start() {
```

Comment:
Space out these methods a bit
```suggestion
        }

        async start() {
```